Assets/Mirage/Runtime/NetworkSceneManager.cs

```diff
@@ -511,8 +514,10 @@ protected internal virtual void OnServerPlayerDisconnected(INetworkPlayer discon
         void HandlePlayerSceneReady(INetworkPlayer player, SceneReadyMessage msg)
         {
             if (logger.LogEnabled()) logger.Log("Default handler for ready message from " + player);
+			
             player.SceneIsReady = true;
-            OnPlayerSceneReady.Invoke(player);
+            
+			OnPlayerSceneReady.Invoke(player);
```

Comment:
Looks ok to me?

Reply:
my editor is showing this as 3 tabs, instead of spaces

Reply:
I use visual studio and resharper. Nothing shows it as tabbed otherwise i would have green lines under it saying its not correct.

![image](https://user-images.githubusercontent.com/11801941/136668393-1aeadb49-133f-412c-b491-06fa2e14b414.png)